Bukkit/src/main/java/com/plotsquared/bukkit/queue/LimitedRegionWrapperQueue.java

```diff
@@ -44,6 +44,7 @@ public class LimitedRegionWrapperQueue extends DelegateQueueCoordinator {
     private static final Logger LOGGER = LogManager.getLogger("PlotSquared/" + LimitedRegionWrapperQueue.class.getSimpleName());
 
     private final LimitedRegion limitedRegion;
+    private boolean useOtherRestoreTagMethod = false;
```

Comment:
Can't we check the minecraft version here directly? (And in that case, the variable could be static final)

Reply:
We don't know that it definitely will or won't fail. I was working on the basis that we don't know for sure if it's an issue that will affect all 1.17.1 builds of spigot/paper